pop/POPAnimator.mm

```diff
@@ -725,6 +770,21 @@ - (id)animationForObject:(id)obj key:(NSString *)key
   return animation;
 }
 
+- (CFTimeInterval)refreshPeriod
+{
+#if TARGET_OS_IPHONE
+  return self->_displayLink.duration;
+#else
+  if (NULL != self->_displayLink) {
+    CVTime period = CVDisplayLinkGetNominalOutputVideoRefreshPeriod(self->_displayLink);
+    if (period.flags & kCVTimeIsIndefinite)
+      return 0;
```

Comment:
Also, a side question - if the display link is invalid (which is the reason to get `kCVTimeIsIndefinite` back here - wouldn't we want to use just a default DisplayTimerFrequency?